lang/cs/Org.Apache.REEF.Driver/Bridge/Events/ActiveContext.cs

```diff
@@ -94,7 +94,7 @@ public void SubmitContextAndService(IConfiguration contextConfiguration, IConfig
 
         public void SendMessage(byte[] message)
```

Comment:
Maybe we need a comment here to tell users that this won't work with C# Evaluators? Also, maybe link to the JIRA for it.